src/consumerIncentives/saga.ts

```diff
@@ -152,6 +152,16 @@ export function* fetchAvailableRewardsSaga({ payload }: ReturnType<typeof fetchA
     return
   }
 
+  const supportedNetworkIds = yield* call(getSupportedNetworkIdsForTokenBalances)
+  const tokensWithTokenBalance = yield* select(tokensWithTokenBalanceSelector, supportedNetworkIds)
```

Comment:
supercharge is only available on Celo. afaik, we don't plan to make it available on Ethereum. or at least, if we do, it will be a good incentive for people to update their apps to participate. so i think it's more correct to only use the celo network id here? (although i think the code here does the bulk of what we need, which is preventing new wallets from fetching supercharge rewards unnecessarily)

Reply:
thanks for highlighting this! sure, it should be Celo only.